code/datums/gas_mixture.dm

```diff
@@ -368,20 +368,20 @@ What are the archived variables for?
 
 	var/delta_temperature = (temperature_archived - model.temperature)
 
-	if(((abs(delta_oxygen) > MINIMUM_AIR_TO_SUSPEND) && (abs(delta_oxygen) >= oxygen_archived*MINIMUM_AIR_RATIO_TO_SUSPEND)) \
```

Comment:
This and the proc, below, as well...they're both disgusting procs that mimic a comparison between a non-simulated turf and a simualted one.